product_custom_info/__openerp__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# © 2015 Antiun Ingeniería S.L. - Carlos Dauden
+# © 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    'name': "Product Custom Info",
+    'summary': "Add custom field in products",
+    'category': 'Customize',
+    'version': '8.0.1.0.0',
+    'depends': [
+        'product',
+        'base_custom_info',
+    ],
+    'data': [
+        'security/ir.model.access.csv',
+        'views/product_view.xml',
+    ],
+    'author': 'Antiun Ingeniería S.L., '
+              'Incaser Informatica S.L., '
+              "Tecnativa, "
+              "Odoo Community Association (OCA)",
+    'website': 'http://www.antiun.com',
+    'license': 'AGPL-3',
+    'installable': True,
```

Comment:
`application` is used for marking a module as a "full-featured" module that allows to use a new area. This can't be considered like that. Some examples of modules with application=True can be the `medical` main module or `donation` module.